### PR TITLE
community/certbot: add certbot-nginx subpackage

### PR DESCRIPTION
**Pull request by xSke**
This adds a package for the nginx plugin for certbot, allowing it to patch your nginx configuration for SSL support.

### CODE REVIEW

**Comment by andypost** [comment]
You need to bump `pkgrel=1`

**Comment by andypost** [comment]
@xSke please squash commits with `git rebase -i HEAD~2`

**Comment by xSke** [comment]
Done @andypost

**Comment by andypost** [comment]
now it needs @fcolista approval

File: community/certbot/APKBUILD [hunk 1]
```diff
@@ -18,7 +18,7 @@ _depends_acme="py-setuptools py-cryptography py-ndg_httpsclient py-asn1 py-opens
 	py-tz py-rfc3339 py-requests py-six py-werkzeug"
 replaces="letsencrypt"
 makedepends="$depends_dev $_depends_acme"
-subpackages="py-acme:acme"
+subpackages="py-acme:acme certbot-nginx:nginx"
```
[the remark this answers is not in the document]
Comment (xSke) [reply community/certbot/APKBUILD]: Can you explain what you mean by the `checkdepends` thing @andypost ?

[the remark this answers is not in the document]
Comment (andypost) [reply community/certbot/APKBUILD]: @xSke I mean that at least `py-setuptools` should not be dependency to `certbot` package, also `py-mock` required only at `check()` time

[the remark this answers is not in the document]
Comment (andypost) [reply community/certbot/APKBUILD]: certbot deps:
```py
install_requires = [
    'acme>=0.26.0',
    # We technically need ConfigArgParse 0.10.0 for Python 2.6 support, but
    # saying so here causes a runtime error against our temporary fork of 0.9.3
    # in which we added 2.6 support (see #2243), so we relax the requirement.
    'ConfigArgParse>=0.9.3',
    'configobj',
    'cryptography>=1.2',  # load_pem_x509_certificate
    'josepy',
    'mock',
    'parsedatetime>=1.3',  # Calendar.parseDT
    'pyrfc3339',
    'pytz',
    'setuptools',
    'zope.component',
    'zope.interface',
]
```
acme deps
```py
install_requires = [
    # load_pem_private/public_key (>=0.6)
    # rsa_recover_prime_factors (>=0.8)
    'cryptography>=0.8',
    # formerly known as acme.jose:
    'josepy>=1.0.0',
    # Connection.set_tlsext_host_name (>=0.13)
    'mock',
    'PyOpenSSL>=0.13',
    'pyrfc3339',
    'pytz',
    'requests[security]>=2.4.1',  # security extras added in 2.4.1
    'requests-toolbelt>=0.3.0',
    'setuptools',
    'six>=1.9.0',  # needed for python_2_unicode_compatible
]
```
cerbot-nginx
```py
install_requires = [
    'acme>=0.26.0',
    'certbot>=0.22.0',
    'mock',
    'PyOpenSSL',
    'pyparsing>=1.5.5',  # Python3 support; perhaps unnecessary?
    'setuptools',
    'zope.interface',
]
```
```

[the remark this answers is not in the document]
Comment (xSke) [reply community/certbot/APKBUILD]: wouldn't this need to go in a separate PR?

**Comment by andypost** [comment]
@xSke needs rebase after 0d21931f

**Comment by fcolista** [comment]
I had not notify for this PR (sorry), therefore I have upgraded certbot to 0.28.0.
So, please rebase the patch, including some changes:
1. that patch should have $pkgname in nginx() depends
2. subpackage too should be $pkgname-nginx:nginx 

Thanks!
.: Francesco

**Comment by clandmeter** [comment]
I just did a big change in certbot to upgrade it to latest version and switch to python 3.
It also includes nginx with proper dependencies.

**Comment by rnalrd** [comment]
Please ask to reopen if clandmeter commit left out something.
Thanks

**Pull request closed without merging**